### PR TITLE
Added api modules to id set

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23160

## Description
Added api_modules to id_set under 'integrations' section.

### CODE REVIEW

**Comment by reutshal** [comment]
@ronykoz I will attach the created id_set soon

File: demisto_sdk/commands/common/update_id_set.py [hunk 1]
```diff
@@ -142,7 +142,17 @@ def get_commmands_from_playbook(data_dict):
     return command_to_integration
 
 
-def get_integration_data(file_path):
+def get_integration_api_modules(file_path, data_dictionary, is_unified_integration):
+    unifier = Unifier(os.path.dirname(file_path))
+    if is_unified_integration:
+        integration_script_code = data_dictionary.get('script', {}).get('script', '')
+    else:
+        _, integration_script_code = unifier.get_script_package_data()
```
Comment (ronykoz) [inline demisto_sdk/commands/common/update_id_set.py:150]: let's update the name of get_script_package_data it no longer works only for scripts

**Comment by reutshal** [comment]
@ronykoz this is the new id_set: https://3724-219291269-gh.circle-artifacts.com/0/content/tmp/id_set.json
You can see the `api_modules` key was added under integration information for every integration that uses one.

File: .circleci/config.yml [hunk 2]
```diff
@@ -37,24 +37,12 @@ jobs:
...
-                exit 0
-              fi
-
-              ./Tests/scripts/validate.sh
-        - run:
-            name: Test create ID set
-            command: |
-              . .tox/py37/bin/activate
-
               if [[ $CIRCLE_BRANCH != master && $FAKE_MASTER != true ]]; then
```
Comment (ronykoz) [inline .circleci/config.yml:46]: why?

**Comment by reutshal** [comment]
@ronykoz this was tested on a master build: https://circleci.com/gh/demisto/demisto-sdk/3768#artifacts/containers/0

**Pull request merged**